src/main/java/hudson/plugins/claim/ClaimColumn.java

```diff
@@ -63,14 +58,12 @@ public boolean shownByDefault() {
                 return result;
     }
 
-    public Descriptor<ListViewColumn> getDescriptor() {
-        return Hudson.getInstance().getDescriptorOrDie(getClass());
+    public ListViewColumnDescriptor getDescriptor() {
```

Comment:
Technically it breaks the binary compatibility. Not sure it's important in this case. Maybe it makes sense to just use `DescriptorImpl` as a return type

Reply:
this is mostly to handle a deprecation on the shownByDefault on the Column itself, to move it to the descriptor class. switched to DescriptorImpl for return value